author/topics/automation/json_to_text.adoc

```diff
@@ -0,0 +1,581 @@
+---
+layout: author-docs
+---
+
+= Json2Text plugin: Content from JSON data
+
+== Purpose
+
+When managing large amounts of data with identical data structures,
+such as registry data or requirements,
+Metanorma supports a templating mechanism for generating structured text
+using data structures as input.
+
+The `json2text` block lets you generate document elements directly
+from a separate JSON file into the document. [added in https://github.com/metanorma/metanorma-standoc/releases/tag/v1.5.3]
+
+== Expressions
+
+`json2text` supports all https://shopify.github.io/liquid/basics/introduction/[Liquid syntax expressions], including:
+
+* variables, variable assignment
+* flow control (if/case)
+* filters
+* loops
+
+See https://shopify.github.io/liquid/basics/introduction/[here] for the full description of Liquid tags and expressions.
+
+
+
+== Syntax
+
+[[defining_syntax]]
+=== Defining the block
+
+A `json2text` block is created with the following syntax.
+
+Block opening and closing is demarcated by an open block syntax (`--`)
+or the `[source]` block syntax (`----` or more `-`).
+
+[source,adoc]
+--
+[json2text,{JSON file path},{self-defined context name}]
+----
+this is content within the json2text block!
+----
+--
+
+Where:
+
+* content within the block is called the "`template`";
+
+* `{JSON file path}` is the location of the JSON file that contains data to be loaded. Location of the JSON file is computed relative to the source directory that `[json2text]` is used (e.g., if `[json2text,data.json,data]` is invoked in an `.adoc` file located at `/foo/bar/doc.adoc`, the data file is expected to be found at `/foo/bar/data.json`);
+
+* `{self-defined context name}` is the name where the JSON data read from the JSON file can be accessed with.
```

Comment:
Sorry, I don't understand this. Do you mean this is the name of the root object that the JSON is parsed into, that Metanorma accesses the data under?

Reply:
Yes, correct. I have the line from yaml2text docs